Source/DafnyLanguageServer/Language/Symbols/FormatExtensions.cs

```diff
@@ -29,7 +30,11 @@ public static class FormatExtensions {
     /// <returns>The text representation of the variable.</returns>
     public static string AsText(this IVariable variable) {
       var ghost = variable.IsGhost ? "ghost " : "";
-      return $"{ghost}{variable.Name}: {variable.Type}";
+      try {
+        return $"{ghost}{variable.Name}: {variable.Type}";
+      } catch (Exception e) {
+        return $"{ghost}{variable.Name}: <Internal error: {e}>";
+      }
```

Comment:
```suggestion
      var ghost = variable.IsGhost ? "ghost " : "";
      string type;
      try {
        type = variable.Type.ToString();
      } catch (Exception e) {
        type = $"<Internal error: {e.Message}>";
      }
      return $"{ghost}{variable.Name}: {type}";
```
Not sure it's really any better, but I really dislike duplicate code. Regardless, calling `Message` instead of `ToString` seems to be the thing to do.